terraform/examples/daos_cluster/variables.tf

```diff
@@ -148,6 +148,7 @@ variable "server_daos_scm_size" {
 variable "server_daos_crt_timeout" {
   description = "crt_timeout"
   default     = 300
+  type        = number
```

Comment:
Added this because pre-commit hook complained that this var was missing a type.

Reply:
yes, it is a good practice to specify types.